### ISSUE TITLE
State of the project?

### ISSUE THREAD

**Issue by matthewr6** [ElementalCode/elemental#204]
Everyone's thoughts so far?  Input?  Want feedback?

I know it was kinda rough at first (e.g. issue #45) but it seems like we're going pretty well, and I'd love to get some input/feedback/more involvement.

**Comment by BookOwl** [comment]
I think that things are going pretty well.

**Comment by towerofnix** [comment]
Same here, looks like things are going quite well (especially in the last few weeks I've noticed)!

**Comment by matthewr6** [comment]
yeah I haven't had any project ideas lately so I started doing stuff here again :P

**Comment by towerofnix** [comment]
Same lol.

**Comment by mrjacobbloom** [comment]
It's looking great, I'm super glad to be back! :dancer:

**Comment by PolyEdge** [comment]
There's that :D thing again ^^^^ :P

**Comment by mrjacobbloom** [comment]
No I wanted a dancer there :package:

**Comment by CosmicWebServices** [comment]
You guys should continue on this
-@IcyCoder

**Comment by PullJosh** [comment]
> You guys should continue on this
-@IcyCoder

Feel free to make a pull request if you'd like! :)
